src/Builder/FrontendDialectTransformer.cpp

```diff
@@ -1445,33 +1445,40 @@ void ImportFrontendModelInternal(onnx::ModelProto &model, MLIRContext &context,
   }
 }
 
-void ImportFrontendModelArray(const void *onnxBuffer, int size,
+// Return 0 on success, error otherwise.
+int ImportFrontendModelArray(const void *onnxBuffer, int size,
     MLIRContext &context, OwningOpRef<ModuleOp> &module,
-    ImportOptions options) {
+    std::string *errorMessage, ImportOptions options) {
   onnx::ModelProto model;
 
   auto parse_success = model.ParseFromArray(onnxBuffer, size);
```

Comment:
Suggest to use `bool` rather than auto here.

Reply:
you caught me there